server/events/templates/policy_check_success_wrapped.tmpl

```diff
@@ -10,5 +10,6 @@
 * :put_litter_in_its_place: To **delete** this plan click [here]({{.LockURL}})
 * :repeat: To re-run policies **plan** this project again by commenting:
     * `{{.RePlanCmd}}`
-</details>{{ template "diverged" . }}
+</details>
+{{.PolicyCheckSummary}}{{ template "diverged" . }}
```

Comment:
renderer parameter contains `PolicyCheckSummary` by https://github.com/runatlantis/atlantis/pull/2452. But template variable is not used, so not shown.